code/modules/shuttle/shuttles/escape_shuttle.dm

```diff
@@ -71,11 +75,78 @@
 		door_handler.control_doors("force-unlock")
 		return
 
+	if(prob(CRASH_LAND_PROBABILITY))
+		create_crash_point()
+	else
+		destination = null
+
 	set_mode(SHUTTLE_IGNITING)
 	on_ignition()
 	setTimer(ignitionTime)
 	launched = TRUE
 
+/obj/docking_port/mobile/escape_shuttle/proc/create_crash_point()
+
+	var/ground_z_level
+
+	var/z_level_index = 1
+	for(var/datum/space_level/level in SSmapping.z_list)
+		if(ZTRAIT_GROUND in level.traits)
+			ground_z_level = z_level_index
+			break
+		z_level_index++
+
+	var/list/potential_areas = SSmapping.areas_in_z[SSmapping.areas_in_z[ground_z_level]]
+
+	var/area/area_picked = potential_areas[rand(1, potential_areas.len)]
+
+	var/list/potential_turfs = list()
+
+	for(var/turf/turf_in_area in area_picked.contents)
+		potential_turfs += turf_in_area
+
+	if(!potential_turfs.len)
+		destination = null
+		return
+
+	var/turf/turf_picked = potential_turfs[rand(1, potential_turfs.len)]
+
+	var/obj/docking_port/stationary/escape_pod/crash_land/temp_escape_pod_port = new(turf_picked)
+	temp_escape_pod_port.width = width
+	temp_escape_pod_port.height = height
+	temp_escape_pod_port.id = id
+
+	destination = temp_escape_pod_port
+
+	crash_land = TRUE
+
+/obj/docking_port/mobile/escape_shuttle/enterTransit()
+	. = ..()
+
+	if(!crash_land)
+		return
+
+	for(var/x_offset = 0 to destination.width)
+		for(var/y_offset = 0 to destination.height)
+			var/turf/closed/wall/found_turf = locate(destination.x + x_offset, destination.y + y_offset, destination.z)
+			if(istype(found_turf) && found_turf.hull)
+				qdel(found_turf)
```

Comment:
What's intent here? You're not supposed to qdel turfs but rather use things like ScrapeAway and ChangeTurf